stdlib/websub/src/main/ballerina/src/websub/publisher_client.bal

```diff
@@ -80,13 +89,17 @@ public type PublisherClient client object {
         }
     }
 
-    # Publishes an update to a remote Ballerina WebSub Hub.
-    #
-    # + topic - The topic for which the update occurred
-    # + payload - The update payload
-    # + contentType - The type of the update content, to set as the `ContentType` header
-    # + headers - The headers, if any, that need to be set
-    # + return - `error` if an error occurred with the update
+# Publishes an update to a remote Ballerina WebSub Hub.
+# ```ballerina
+# error? publishUpdate = websubHubClientEP->publishUpdate("http://websubpubtopic.com",{"action": "publish",
+#                                                          "mode": "remote-hub"});
+#  ```
+#
+# + topic - The topic for which the update occurred
+# + payload - The update payload
+# + contentType - The type of the update content, to set as the `ContentType` header
+# + headers - The headers, if any, that need to be set
+# + return -  An `error`if an error occurred with the update or else `()`
```

Comment:
```suggestion
# + return -  An `error`if an error occurred with the update or else `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.